### ISSUE TITLE
Remove facility_locator_show_community_cares flipper

### ISSUE THREAD

**Issue by ryguyk** [department-of-veterans-affairs/va.gov-cms#14112]
## Update 1/15/2025

[Randi Mays]: This flipper is not used meaningfully in vets-website. It can be removed from code wherever it is used.

## Description
See [discovery ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11598). This is a follow-up ticket to actually perform the Flipper removal.

There is work that needs to be done in two repos:
1. vets-api
   - As described in [discovery findings](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11598#issuecomment-1595761028), this Flipper feature flag needs to be replaced by another suitable candidate.
   - We need a Platform support ticket to see who should own that. It feels like Platform should be making that change since the testing that is using this Flipper feature flag is not actually a Facilities' concern. 
       - [Plat support ticket](https://dsva.slack.com/archives/CBU0KDSB1/p1687275578072799): From Brandon Dech: "I've reached out to my team on this request. We don't want to delete the test file altogether because it's being used elsewhere. I think we should opt for the solution of replacing with an alternate flipper. What that alternate flipper might be I'm uncertain."

2. vets-website
   - This work has been done and is in a [draft PR](https://github.com/department-of-veterans-affairs/vets-website/pull/24593).
   - The PR needs to be finalized and, ultimately, merged.

## Acceptance Criteria
- [ ] ~Platform support request is submitted.~
- [ ] ~Outcome of Platform support request is documented in this ticket.~
- [ ] ~If Platform requests we implement, a new ticket is created for the vets-api implementation.~
- [ ] ~vets-website PR is opened (currently in draft) and merged.~
- [x] 1. Remove `facility_locator_show_community_cares` flipper and code implementation from vets-website
- [x] 2. Turn off flipper in all Flipper Admin environments
- [x] 3. Remove flipper from vets-api
- [x] 4. Once the vets-api PR has been merged, notify Platform that the feature flipper should be removed from the flipper dashboards in staging and production. ([example](https://dsva.slack.com/archives/CBU0KDSB1/p1725555324327549))

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [ ] `Public Websites`
- [x] `Facilities`
- [ ] `User support`

**Comment by jilladams** [comment]
Until Plat tells us what to use instead in their test, we are technically blocked. However: if we decide this needs to move along, we could pick one of our own flippers that we know can't be removed, and list that instead. Not urgent.

**Comment by jilladams** [comment]
Test dependency is removed: https://github.com/department-of-veterans-affairs/vets-api/pull/13069/files

**Comment by ryguyk** [comment]
> Test dependency is removed: https://github.com/department-of-veterans-affairs/vets-api/pull/13069/files

This Flipper is still being used in [another test file](https://github.com/department-of-veterans-affairs/vets-api/blob/ea27431db642ea0ab35b3e042f797d82626c4f4a/spec/lib/flipper/instrumentation/event_subscriber_spec.rb).

**Comment by jilladams** [comment]
Kicked that back to Plat here; https://github.com/department-of-veterans-affairs/va.gov-team/issues/60516#issuecomment-1603203487

**Comment by jilladams** [comment]
Possible to triple check that's the full list of places it's used / needs to be replaced?

**Comment by ryguyk** [comment]
> Possible to triple check that's the full list of places it's used / needs to be replaced?

The only two places a reference to `facility_locator_show_community_cares` remains are:
1. The [file noted above](https://github.com/department-of-veterans-affairs/vets-api/blob/ea27431db642ea0ab35b3e042f797d82626c4f4a/spec/lib/flipper/instrumentation/event_subscriber_spec.rb).
2. [`features.yml`](https://github.com/department-of-veterans-affairs/vets-api/blob/5050879892a9b3474bcd5b81227940c2732ccee4/config/features.yml#L338-L343), where the Flipper flag is defined. This can be removed once the test files remove references.

**Comment by jilladams** [comment]
Plat completed removing the other reference so this should be actually unblocked now.

**Comment by randimays** [comment]
Flipper is removed from all code and the flipper dashboards in all environments. Closing as complete.
